website/addons/osfstorage/routes.py

```diff
@@ -112,6 +112,24 @@
             views.osfstorage_copy_hook,
             json_renderer,
         ),
+
+        Rule(
+            [
+                '/project/<pid>/osfstorage/<fid>/tags/<tag>/',
```

Comment:
My main question: does this have to be done with the v1 api?

But Steve is correct, in the body is where that should be. Also, we're going to want to have the mechanism be the same as for project tags when we put it in v2, so having a list that's always passed rather than treating it as a collection (or really quickly refactoring node tags to work more like a collection, but that seems super-high overhead for tags)